src/frontend/js/contexts/OrderContext/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { createContext, PropsWithChildren, useContext, useEffect, useMemo, useState } from 'react';
+import { Order } from '../../types/Joanie';
+import { Maybe } from '../../types/utils';
+
+export interface OrderContextInterface {
+  currentOrder: Maybe<Order>;
+  setCurrentOrder: (order: Maybe<Order>) => void;
+}
+
+export const OrderContext = createContext<Maybe<OrderContextInterface>>(undefined);
+
+export const useCurrentOrderContext = () => {
```

Comment:
By convention, this kind of hook are always names with this pattern : 
`use<ContextName>`
```suggestion
export const useOrderContext = () => {
```